src/sentry/tasks/integrations.py

```diff
@@ -51,3 +51,43 @@ def sync_assignee_outbound(external_issue_id, user_id, assign, **kwargs):
     else:
         user = User.objects.get(id=user_id)
     integration.get_installation().sync_assignee_outbound(external_issue, user, assign=assign)
+
+
+@instrumented_task(
+    name='sentry.tasks.integrations.sync_status_outbound',
+    queue='integrations',
+    default_retry_delay=60 * 5,
+    max_retries=5
+)
+@retry(exclude=(ExternalIssue.DoesNotExist, Integration.DoesNotExist))
+def sync_status_outbound(external_issue_id, is_resolved, **kwargs):
+    external_issue = ExternalIssue.objects.get(id=external_issue_id)
+    integration = Integration.objects.get(id=external_issue.integration_id)
+    integration.get_installation().sync_status_outbound(external_issue, is_resolved)
+
+
+@instrumented_task(
+    name='sentry.tasks.integrations.kick_off_status_syncs',
+    queue='integrations',
+    default_retry_delay=60 * 5,
+    max_retries=5
+)
+@retry(exclude=(Group.DoesNotExist,))
+def kick_off_status_syncs(group_id, is_resolved, **kwargs):
+    # doing this in a task since this has to go in the event manager
+    # and didn't want to introduce additional queries there
+    group = Group.objects.get(id=group_id)
+    if features.has('organizations:internal-catchall', group.organization):
```

Comment:
nit: This would probably look a little nicer as a guard statement

```py
if not features.has('organizations:internal-catchall', group.organization):
    return
```